_data/chains/eip155-2001.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Milkomeda C1 Mainnet",
+  "chain": "milkAda",
+  "icon": "milkomeda",
+  "rpc": [
+    "https://rpc.c1.milkomeda.com:8545",
+    "wss://rpc.c1.milkomeda.com:8546"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "MilkAda",
```

Comment:
can we keep it with a non-capitalized `m` 🙏 `milkADA`

Reply:
Of course @nicarq, I'll update it shortly

Reply:
can you change the `testnet` one too please @bartfish  ?

Reply:
ah yes, I'll update this one shortly too